packages/paste-website/src/pages/core/libraries/code-editor/index.mdx

```diff
@@ -0,0 +1,108 @@
+export const meta = {
+  title: 'Code Editor',
+  package: '@twilio-paste/code-editor-library',
+  description: 'A library to display and make changes to large blocks of code.',
+  slug: '/core/libraries/code-editor/',
+};
+
+import packageJson from '@twilio-paste/code-editor-library/package.json';
+import {SidebarCategoryRoutes} from '../../../../constants';
+import {getNavigationData} from '../../../../utils/api';
+import DefaultLayout from '../../../../layouts/DefaultLayout';
+
+export default DefaultLayout;
+
+export const getStaticProps = async () => {
+  const navigationData = await getNavigationData();
+  return {
+    props: {
+      navigationData,
+    },
+  };
+};
+
+<GenericHeader
+  name={meta.title}
+  categoryRoute={SidebarCategoryRoutes.LIBRARIES}
+  storybookUrl="/?path=/story/libraries-code-editor-editor--"
+  githubUrl="https://github.com/twilio-labs/paste/tree/main/packages/paste-libraries/code-editor"
+  packageName={packageJson.name}
+  version={packageJson.version}
+  description={packageJson.description}
+/>
+
+---
+
+<contentwrapper>
+<PageAside data={mdxHeadings} />
+<content>
+
+## About
+
+The Code Editor allows for a full code-editing experiences on the web, including syntax highlighting and displaying line numbers.
+
+This library is built on top of [monaco-react](https://github.com/suren-atoyan/monaco-react), which wraps the
+[Monaco Editor](https://github.com/microsoft/monaco-editor).
+
+Our library exposes a theme for use in Twilio products, aptly called the `PasteTheme`. This theme is based on the
+`night owl` theme.
```

Comment:
```suggestion
Our library exposes a theme for use in Twilio products, aptly called the `PasteTheme`. This theme is based on the
[Night Owl theme by Sarah Drasner](Prob this link instead because night owl isn't mentioned in the CSS Tricks article https://marketplace.visualstudio.com/items?itemName=sdras.night-owl&WT.mc_id=github-theme-sdras).
```